content/applications/finance/accounting/customer_invoices/deferred_revenues.rst

```diff
@@ -1,195 +1,206 @@
-   Revenues` and then, by clicking on the button *Save Model*.
+- Entry 2 dated on the 1st March, the reversal of the previous entry.
 
-Apply a Deferred Revenue Model to a new entry
----------------------------------------------
+From March to October
+~~~~~~~~~~~~~~~~~~~~~
 
-When you create a new Deferred Revenue entry,  fill out the **Deferred Revenue Account** with the
-right recognition account.
+The same computation is done for each month until October.
 
-New buttons with all the models linked to that account appear at the top of the form. Clicking on a
-model button fills out the form according to that model.
+In November
+~~~~~~~~~~~
 
-.. image:: deferred_revenues/deferred_revenues08.png
-   :align: center
-   :alt: Deferred Revenue model button in Odoo Accounting
+At the end of November, after clicking the :guilabel:`Generate Entries` button, we will have the following entries:
 
-.. _deferred-revenues-automation:
+- Entry 1 dated on the 30th November:
 
-Automate the Deferred Revenues
-==============================
+  - Line 1: Expense account -1200 -600 = **-1800** (we cancel the total of both invoices)
+  - Line 2: Expense account 1100 + 550 = **1650** (we recognize 11/12 of invoice A and 11/12 of invoice B)
+  - Line 3: Deferred account 1800 - 1650 = **150** (amount that has yet to be deferred later on)
 
-When you create or edit an account of which the type is either *Current Liabilities* or *Non-current
-Liabilities*, you can configure it to defer the revenues that are credited on it automatically.
+- Entry 2 dated on the 1st December, the reversal of the previous entry.
 
-You have three choices for the **Automate Deferred Revenue** field:
+In December
+~~~~~~~~~~~
 
-#. **No:** this is the default value. Nothing happens.
-#. **Create in draft:** whenever a transaction is posted on the account, a draft *Deferred Revenues
-   entry* is created, but not validated. You must first fill out the form in
-   :menuselection:`Accounting --> Accounting --> Deferred Revenues`.
-#. **Create and validate:** you must also select a Deferred Revenue Model (see:
-   `Deferred Revenue Models`_). Whenever a transaction is posted on the account, a *Deferred
-   Revenues entry* is created and immediately validated.
+There is no need to generate entries in December. Indeed, if we do the computation for December,
+we will have an amount of 0 to be deferred.
 
-.. image:: deferred_revenues/deferred_revenues09.png
-   :align: center
-   :alt: Automate Deferred Revenue on an account in Odoo Accounting
+In total
+~~~~~~~~
+
+If we aggregate everything, we would have:
+
+- invoice A and invoice B
+- 2 entries (one for the deferral and one for the reversal) for each month from January to November
+
+Therefore, at the end of December, invoices A and B are fully recognized as income,
+only once thanks to the reversal mechanism.
+
+Why would you need the "Manually & Grouped" mode?
+-------------------------------------------------
+
```

Comment:
This section is very useful, but we usually avoid FAQ sectionsin the documentation. Instad let's put this in a tip admonition block (.. tip::) and remove this line:
```suggestion
.. tip::
```